test/e2e/arktos/multi_tenancy/test_suites/kubectl/test_autoscale.yaml

```diff
@@ -0,0 +1,46 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Kubectl autoscale Tests ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies command "kubectl autoscale".
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  test_tenant: random_8
+  test_ns: random_8
+
+###########################################################################################################
+# test setup
+###########################################################################################################
+Tests:
+  - BeforeTestMessage: Starting test setup ...
+    Command: ${kubectl} create tenant ${test_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${test_tenant} created\n"
+
+  - Command: ${kubectl} create namespace ${test_ns} --tenant ${test_tenant}
+    OutputShouldBe: "namespace/${test_ns} created\n"
+
+  - Command:  ${kubectl} apply -f ${test_data_dir}/sample-deployment.yaml -n ${test_ns} --tenant ${test_tenant} 
+    OutputShouldBe: "deployment.apps/sample-nginx-deployment created\n"
+
+############################################
+# test "kubectl autoscale"
+############################################
+  - BeforeTestMessage: "Verifying \"kubectl autoscale\" ..."
+    Command: ${kubectl} get horizontalpodautoscalers -n ${test_ns} --tenant ${test_tenant} 
+    OutputShouldBe: "No resources found.\n"
+
+  - Command: ${kubectl} autoscale deployment sample-nginx-deployment --cpu-percent=50 --min=1 --max=5 -n ${test_ns} --tenant ${test_tenant}
+    OutputShouldBe: "horizontalpodautoscaler.autoscaling/sample-nginx-deployment autoscaled\n"
+
+  - Command: "${kubectl} get horizontalpodautoscalers sample-nginx-deployment -n ${test_ns} --tenant ${test_tenant} -o json
+              | jq -r '[.metadata.name, .metadata.namespace, .metadata.tenant, .spec.maxReplicas, .spec.minReplicas, .spec.targetCPUUtilizationPercentage] | @tsv'"
+    OutputShouldBe: "sample-nginx-deployment	${test_ns}	${test_tenant}	5	1	50\n"
```

Comment:
the spaces here look unusual. looks like some are \t and some are spaces.

Reply:
They are all "\t" actually. Otherwise the test will not pass, as the command output is controlled by  "| @tsv".